kroxylicious-api/src/test/java/io/kroxylicious/proxy/config/tls/TlsTest.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.proxy.config.tls;
+
+import java.security.KeyStore;
+import java.util.Locale;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class TlsTest {
+
+    @Test
+    void testGetStoreTypeOrPlatformDefault() {
+        assertThat(Tls.getStoreTypeOrPlatformDefault("PEM")).isEqualTo("PEM");
+        assertThat(Tls.getStoreTypeOrPlatformDefault("JKS")).isEqualTo("JKS");
+        assertThat(Tls.getStoreTypeOrPlatformDefault("jKs")).isEqualTo("JKS");
+        assertThat(Tls.getStoreTypeOrPlatformDefault("jks")).isEqualTo("JKS");
```

Comment:
should we be testing other capitalisations of `PEM`?

Reply:
the input is arbitrary as it uppercases whatever you give it.

Reply:
cool it just seemed strange to see a mix of cases for one name and not the other...